go.sum

```diff
@@ -299,6 +297,8 @@ github.com/bombsimon/wsl/v2 v2.0.0/go.mod h1:mf25kr/SqFEPhhcxW1+7pxzGlW+hIl/hYTK
 github.com/bombsimon/wsl/v2 v2.2.0/go.mod h1:Azh8c3XGEJl9LyX0/sFC+CKMc7Ssgua0g+6abzXN4Pg=
 github.com/bombsimon/wsl/v3 v3.0.0/go.mod h1:st10JtZYLE4D5sC7b8xV4zTKZwAQjCH/Hy2Pm1FNZIc=
 github.com/bombsimon/wsl/v3 v3.1.0/go.mod h1:st10JtZYLE4D5sC7b8xV4zTKZwAQjCH/Hy2Pm1FNZIc=
+github.com/briandowns/spinner v1.16.0 h1:DFmp6hEaIx2QXXuqSJmtfSBSAjRmpGiKG6ip2Wm/yOs=
+github.com/briandowns/spinner v1.16.0/go.mod h1:QOuQk7x+EaDASo80FEXwlwiA+j/PPIcX3FScO+3/ZPQ=
```

Comment:
Could we pick up latest release https://github.com/briandowns/spinner/releases/tag/v1.18.0?

Reply:
The latest release would clean up the line after the spinner is stopped. https://github.com/briandowns/spinner/commit/12b31a684908a81e40889585a0fb69a5313a7eba#diff-7fa6384fd96c30fe4860e14a0f0d77214ebcc6af75b809031f21eb6a1cf0b1c7

Reply:
Could we print a new line before spinner is stopped?

Reply:
Actually, is it better to just eliminate each line after spinner stops? The spinner message will be shown during the process. The following are 2 types of output.

```
$ ./tanzu-core-darwin_amd64 package install my-ingress1 -p contour.tanzu.vmware.com -v 1.18.2+vmware.1-tkg.1-rc.4

Updated installed package 'my-ingress1'
```

```
$ ./tanzu-core-darwin_amd64 package install my-ingress1 -p contour.tanzu.vmware.com -v 1.18.2+vmware.1-tkg.1-rc.4
 Installing package 'contour.tanzu.vmware.com'
 Updating package 'my-ingress1'
 Getting package install for 'my-ingress1'

Updated installed package 'my-ingress1'
```

Reply:
NVM, keeping the message looks better